frontend/modules/api/controllers/MessagesController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace frontend\modules\api\controllers;
+
+use frontend\models\Messages;
+use Yii;
+use yii\rest\ActiveController;
+
+class MessagesController extends ActiveController
+{
+    public $modelClass = Messages::class;
+
+    public function actions()
+    {
+        $actions = parent::actions();
+
+        unset($actions['view'], $actions['create']);
+
+        return $actions;
+    }
+
+    public function actionCreate($id)
+    {
+        $content = json_decode(Yii::$app->getRequest()->getRawBody());
+        $message = new Messages();
+        $message->user_id = Yii::$app->user->getId();
+        $message->task_id = $id;
+        $message->message = $content->message;
+        $message->save();
+    }
```

Comment:
В дополнение к 201 коду, что я указал выше, тут нужно вернуть json нового сообщения, чтобы его отобразить:
```suggestion
        return $message;
    }
```